components/Article/Page.js

```diff
@@ -327,11 +332,26 @@ const ArticlePage = ({
   const hasOverviewNav = meta ? meta.template === 'section' : true // show/keep around while loading meta
   const colorSchemeKey = darkMode ? 'dark' : 'auto'
 
+  const shareImage = `${ASSETS_SERVER_BASE_URL}/render?width=${SHARE_IMAGE_WIDTH}&height=${SHARE_IMAGE_HEIGHT}&updatedAt=${encodeURIComponent(
+    article.id
```

Comment:
ensure image gets re-downloaded by FB & co. if the background or color is changed in the format:
```suggestion
    `${article.id}${meta?.format ? `-${meta.format.id}` : ''}`
```